### ISSUE TITLE
using a newline in the left prompt causes the right prompt to start on the second line

### ISSUE THREAD

**Issue by martmists** [Powerlevel9k/powerlevel9k#1363]
#### Describe Your Issue
The right prompt is on the second line due to the `newline` segment in the left prompt
![image](https://user-images.githubusercontent.com/16361449/64961489-e9750d80-d895-11e9-8989-b7f97612ba02.png)

#### Have you tried to debug or fix it?

I have tried to make a manual newline prompt and use terminal tweaks, but couldn't find a solution.

#### Environment Information

  - ZSH 5.7.1
  - P9K installed from AUR, latest version
  - Termite VTE

**Comment by MaxL** [comment]
you can use `POWERLEVEL9K_RPROMPT_ON_NEWLINE=false` for that

**Comment by romkatv** [comment]
`POWERLEVEL9K_RPROMPT_ON_NEWLINE=false` is the default, so it won't do anything if you set it explicitly.

The problem is that `POWERLEVEL9K_RPROMPT_ON_NEWLINE` is ignored unless `POWERLEVEL9K_PROMPT_ON_NEWLINE` is set to `true`. However, the latter means that you cannot put any prompt segments on the last line. There is no fundamental reason for these restrictions. They just came about.

**Comment by martmists** [comment]
So are there any plans to fix this bug?

**Comment by romkatv** [comment]
I'm not a powerlevel9k dev (nor a user), so I wouldn't know what the plans are. There is [not much activity](https://github.com/Powerlevel9k/powerlevel9k/graphs/commit-activity) right now, so I wouldn't bet on this issue being resolved soon.

P.S.

I think calling it a limitation rather than a bug is more appropriate. It doesn't help you, of course, but it's more charitable to the devs.

**Comment by martmists** [comment]
Seems like this is a known bug and #843 will fix this, if merged.

**Comment by romkatv** [comment]
> Seems like this is a known bug and #843 will fix this, if merged.

Yep. It cannot be merged into `master` though, for it'll break the majority of p9k configurations, including the default. It could be merged into `next`, where it's sorta OK to introduce non-backward-compatible changes.

Once again, I'm not a powerlevel9k dev, so my opinion doesn't carry any weight here.
